docs/src/main/asciidoc/logging.adoc

```diff
@@ -125,6 +125,8 @@ This setting is only evaluated if `includeFormattedMessage` is `true`
 | `includeContextName` | `true` | Should the logging context be included
 | `includeMessage` | `false` | Should the log message with blank placeholders be included
 | `includeException` | `false` | Should the stacktrace be included as a own field
+| `serviceContext` | none | Define the service context data.
+| `customJson` | none | Defines custom json data. Data will be added to the json output.
```

Comment:
Can you explain the use-case for this? You already have the MDC to provide attributes.

Reply:
I think the option to put some custom static data is helpful for some custom use cases. We use this for example  to add some environment variables from the docker container to the json payload. So we can search and filter for these metadata. To do this with MDC has disadvantages: you have to write code to just add some static json payload. With MDC you can only add String / String payload. With this new option, you only need to add the json to the logback configuration.